softdev.bib

```diff
@@ -2899,3 +2899,70 @@ @inproceedings{shahriyar14fast
     month = Oct,
     pages = "121-139"
 }
+
+@article{martin08common,
+  author = {Martin, Robert A. and Barnum, Sean},
+  title = {Common Weakness Enumeration (CWE) Status Update},
+  year = {2008},
+  issue_date = {April 2008},
+  publisher = {Association for Computing Machinery},
+  address = {New York, NY, USA},
+  volume = {XXVIII},
+  number = {1},
+  issn = {1094-3641},
+  url = {https://doi.org/10.1145/1387830.1387835},
+  doi = {10.1145/1387830.1387835},
+  journal = {Ada Lett.},
```

Comment:
Probably "Ada Letters" is better?

Reply:
Fixed in https://github.com/softdevteam/softdevbib/pull/86/commits/d48939d804f587f9f946f4a0f4fc2b1d69c69820.